app/src/state_machine.cpp

```diff
@@ -104,17 +117,10 @@ void hvacStateUpdate()
   HVAC_MODE prev_mode = OperatingParameters.hvacOpMode;
 
   currentTemp = OperatingParameters.tempCurrent + OperatingParameters.tempCorrection;
-  minTemp = OperatingParameters.tempSet - (OperatingParameters.tempSwing / 2.0);
-  maxTemp = OperatingParameters.tempSet + (OperatingParameters.tempSwing / 2.0);
-
-// The following code is for later when auto mode is supported fully
-#if 0
```

Comment:
This code will eventually be used (when auto mode is completely implemented). Using OperatingParameters.tempSwing is necessary for auto mode.

Reply:
The code isn't thrown away, though. Just hidden under AUTOMODE_SUPPORTED within the wrappers.